data/test/tabletserver/ddl_cases.txt

```diff
@@ -1,86 +1,86 @@
 "create table a(abcd)"
 {
-  "Action": "create", "NewName": "a"
+  "Action": "create"
 }
 
 "drop  table b"
 {
-  "Action": "drop", "TableName": "b"
+  "Action": "drop"
```

Comment:
This test should be improved. Right now, it passes with both versions of the expected result (without the code changes in this PR). It should have a single expected result. Also the attribute names are irrelevant. I could change this to `"Action": "drop", "XYZ": "b"` and it will pass. This makes it confusing to someone who is trying to correlate the test with the code.

The expected result should be `"Action": "drop", "FromTables": "b"`